Update dependencies

Abban opened the pull request:
Skips @typescript-eslint/eslint-plugin for now until
the peer dependencies sort themselves out.

moiikana commented on shared/components/ui/ProgressBar.jsx:
```diff
@@ -107,9 +107,6 @@ export default class ProgressBar extends Component {
 		const Translations = context;
 		const getMillion = n => this.props.formatters.millionFormatter( n / 1000000 );
 		const getDaysLeft = daysLeft => {
-			if ( props.daysLeft > 14 ) {
```
not so sure if it's fine to delete this in the shared progress bar, but maybe you've already talked about that

Abban replied:
No we didn't talk about it at all! I think that we should be manually hiding the text with css anyway instgead of having it automatically appear.

We actually should check the existing banners that are going to have that text appear at the weekend and make sure it's styled though!

moiikana reviewed (changes requested):
might be missing the footer

Abban committed:
Update dependencies

Skips @typescript-eslint/eslint-plugin for now until
the peer dependencies sort themselves out.
